openapi-generator/src/main/kotlin/org/jellyfin/openapi/builder/openapi/OpenApiApiServicesBuilder.kt

```diff
@@ -73,12 +74,19 @@ class OpenApiApiServicesBuilder(
 				if (returnType == Unit::class.asTypeName() && "200" in operation.responses)
 					println("Missing return-type for operation $operationName (status-codes: ${operation.responses.keys})")
 
+				val requireAuthentication = operation.security
+					?.mapNotNull { it[Security.SECURITY_SCHEME] }
+					?.firstOrNull()
+					?.any(Security.AUTHENTICATION_POLICIES::contains)
+					?: false
```

Comment:
I don't really like this but unfortunately the data structure is awful. It's a `List<Map<String, List<String>>>`. Feel free to suggest a better approach.

Reply:
What about that?
```kt
				val requireAuthentication = operation.security
					?.firstOrNull { requirement -> requirement.containsKey(Security.SECURITY_SCHEME) }
					?.get(Security.SECURITY_SCHEME)
					?.any { policy -> policy in Security.AUTHENTICATION_POLICIES }
					?: false
```

Reply:
Ended up with 
```kt
val requireAuthentication = operation.security
	?.firstOrNull { requirement -> requirement.containsKey(Security.SECURITY_SCHEME) }
	?.get(Security.SECURITY_SCHEME)
	?.any(Security.AUTHENTICATION_POLICIES::contains)
	?: false
```

Reply:
I thought the `in` would be more readable, but it's fine this way.

Reply:
I'm quite the fan of function references haha

Reply:
Same! But in some cases the readability does suffer a bit. It's ok here, so I don't worry too much.